### PR TITLE
Add a document for development

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Setup
+
+Please install [yarn](https://yarnpkg.com/en/docs/install) and [hug](https://gohugo.io/getting-started/installing/) first.
```
Comment (vdemeester) [inline DEVELOPMENT.md]: there is a typo here (`s/hug/hugo`)

Comment (LinuxSuRen) [reply DEVELOPMENT.md]: Thanks for reminding.

**Comment by bobcatfish** [comment]
Whoops @LinuxSuRen looks like I made some changes that conflict with yours :(

**Comment by LinuxSuRen** [comment]
The latest version already covered mine. I'm going to close this one.

**Comment by bobcatfish** [comment]
kk sounds good - thanks and sorry for stomping over your changes @LinuxSuRen !!

**Comment by LinuxSuRen** [comment]
No worries.

**Pull request closed without merging**